2022q2/azure.adoc

```diff
@@ -0,0 +1,39 @@
+=== FreeBSD on Microsoft HyperV and Azure
+
+Links: +
+link:https://wiki.freebsd.org/MicrosoftAzure[Microsoft Azure article on FreeBSD wiki] URL: link:https://wiki.freebsd.org/MicrosoftAzure[] +
+link:https://wiki.freebsd.org/HyperV[Microsoft HyperV article on FreeBSD wiki] URL: link:https://wiki.freebsd.org/HyperV[]
+
+Contact: Microsoft FreeBSD Integration Services Team <bsdic@microsoft.com> +
+Contact: link:https://lists.FreeBSD.org/mailman/listinfo/freebsd-cloud[freebsd-cloud Mailing List] +
+Contact: The FreeBSD Azure Release Engineering Team <releng-azure@FreeBSD.org> +
+Contact: Wei Hu <whu@FreeBSD.org> +
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org> +
+
+The link:https://azuremarketplace.microsoft.com/marketplace/apps/thefreebsdfoundation.freebsd-13_1[13.1-RELEASE image on Azure Marketplace] has been published.
+
+Work in progress tasks:
+
+* Automatizing the image building and publishing process
+* Build and publish ZFS-based image to Azure Marketplace
+** The taks will be benefited by merging of ZFS support of makefs(8) and release(7)
+*** https://reviews.freebsd.org/D23334
+*** https://reviews.freebsd.org/D34426
+*** https://reviews.freebsd.org/D35248
+* Build and publish gen2 vm image to Azure Marketplace
+** Blocked by https://bugs.freebsd.org/264267
+
+Above tasks are sponsored by The FreeBSD Foundation, with resources provided by Microsoft.
+
+Wei Hu and his colleagues in Microsoft are working on several tasks sponsored by Microsoft:
```

Comment:
Noting that sponsoring is usually at the end of the section. See report-sample.adoc.

Reply:
Yeah, but those items are sponsored by different entities so I want to be more explicit.

Reply:
> Yeah, but those items are sponsored by different entities so I want to be more explicit.

Left it and added a summary in the usual location.